app/controllers/concerns/accessible.rb

```diff
@@ -0,0 +1,20 @@
+module Accessible
+    extend ActiveSupport::Concern
+    included do
+      before_action :check_user
+    end
+  
+    protected
+    def check_user
+      if current_admin
+        flash.clear
+        # if you have rails_admin. You can redirect anywhere really
+        redirect_to(rails_admin.dashboard_path) and return

```

Comment:
Did you use `rails_admin` ?

Reply:
No Maam, will resolve in the next commit